src/engines/html5/media-source/adapters/native-adapter.js

```diff
@@ -249,15 +258,15 @@ export default class NativeAdapter extends BaseMediaSourceAdapter {
 
   /**
    * Get the parsed tracks
-   * @function _getParsedTracks
-   * @returns {Array<Track>} - The parsed tracks
+   * @function _parseTracks
+   * @returns {void}
    * @private
    */
-  _getParsedTracks(): Array<Track> {
-    let videoTracks = this._getParsedVideoTracks();
-    let audioTracks = this._getParsedAudioTracks();
-    let textTracks = this._getParsedTextTracks();
-    return videoTracks.concat(audioTracks).concat(textTracks);
+  _parseTracks(): void {
```

Comment:
why not return tracks from here? we do the same in HLS and DASH adapters so why change convention?

Reply:
done